app/controllers/user_controller.rb

```diff
@@ -0,0 +1,23 @@
+require './lib/user'
+require_relative 'app'
+
+class UserController < Sinatra::Base 
```

Comment:
Really cool how you've separated this into two controllers, it has really added a lot.